packages/react/src/components/ModalWrapper/ModalWrapper.js

```diff
@@ -10,8 +10,19 @@ import React from 'react';
 import Modal from '../Modal';
 import Button from '../Button';
 import { ButtonKinds } from '../../prop-types/types';
+import { warning } from '../../internal/warning';
+
+let didWarnAboutDeprecation = false;
 
 export default class ModalWrapper extends React.Component {
+  if(__DEV__) {
+    warning(
+      didWarnAboutDeprecation,
+      '`<ModalWrapper>` has been deprecated in favor of `<ComposedModal/>` and will be removed in v12 of `@carbon/react`'
```

Comment:
Since this is a message to devs, should we clarify the version number to match the real version number of the package?

```suggestion
      '`<ModalWrapper>` has been deprecated in favor of `<ComposedModal/>` and will be removed in the next major version, `@carbon/react@v2.x`'
```

Reply:
that makes sense 👍🏽